pom.xml

```diff
@@ -1013,6 +1014,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>7</source>
+                    <target>7</target>
+                </configuration>
+            </plugin>
```

Comment:
Why was this needed?

Reply:
going to blame IDEA here, I think it added it when I marked the jbang dir as a source root :facepalm: